server.js

```diff
@@ -8,13 +8,52 @@ app.use(cors());
 
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
-
 app.get("/", function (request, response) {
-  response.send("Hotel booking server.  Ask for /bookings, etc.");
+  response.send({ bookings });
 });
+//Read all bookings
 
+app.get("/booking", function (request, response) {
+  response.send({ bookings });
+});
+//Create a new booking
+app.post("/booking", (req, resp) => {
+  const newBooking = req.body;
+  console.log(newBooking);
+  if (
+    newBooking.id &&
+    newBooking.title &&
+    newBooking.firstName &&
+    newBooking.surname &&
+    newBooking.email &&
+    newBooking.roomId &&
+    newBooking.checkInDate &&
+    newBooking.checkOutDate
+  ) {
+    bookings.push(newBooking);
+    resp.send("New Booking has been added >>");
```

Comment:
In the response message, you could also return the values from the array to make sure you have updated them.